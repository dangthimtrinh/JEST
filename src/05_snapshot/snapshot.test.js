describe("snapshot test", () => {
    let tree;

    beforeEach(() => {    
        tree = "<div>Facebook</div>";
    });

    it('correctly', () => {
        expect(tree).toMatchSnapshot();
    });
});
