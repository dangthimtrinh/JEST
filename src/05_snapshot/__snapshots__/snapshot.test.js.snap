// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test correctly 1`] = `"<div>Instagram</div>"`;
